kyo-bench/src/main/scala/kyo/bench/HttpClientRaceContentionBench.scala

```diff
@@ -1,81 +0,0 @@
-// package kyo.bench
```

Comment:
I tried to fix this benchmark. I got cats and kyo working but zio has some odd failures, I suspect some bug. I've removed the benchmark instead. The comparison isn't great anyway given the custom code for cats-effect